rustfmt.toml

```diff
@@ -1,43 +0,0 @@
-# Specifies which edition is used by the parser.
```

Comment:
not sure this is a good idea to remove this file: it tends to make sure than everyone is not free to format rust code as he wants

Reply:
1 good feature of the file is to reorder `imports` always in the same way